### PR TITLE
Add default options repo in worldview

### PR DESCRIPTION
**Pull request by ZachTRice**
Connects to #507 
Connects to #512
- Added worldview-options to package.json with version tag.
- Modified build and grunt scripts to check the options directory; if there are custom options, use these. If not, use the worldview-options-eosdis package located in the node_modules folder.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -45,16 +45,28 @@ module.exports = function(grunt) {
 
   var env = grunt.option("env") || "release";
 
+  var pkg = grunt.file.readJSON("package.json");
```
Comment (localjo) [inline Gruntfile.js]: Should use `var pkg = require('package.json')` at the top of the file instead.

File: Gruntfile.js [hunk 2]
```diff
@@ -45,16 +45,28 @@ module.exports = function(grunt) {
...
+
+  var optionsPath;
+
+  if (fs.existsSync("options")) {
+    optionsPath = "options";
+  }
+  else if (fs.existsSync("node_modules/worldview-options-eosdis")) {
```
Comment (localjo) [inline Gruntfile.js]: Lines 50-58 could just be;
`var optionsPath = hasOptionsDirectory? 'options' : 'node_modules/worldview-options-eosdis';`

And I'd move that up to the top of the file right after setting `hasOptionsDirectory`.

File: Gruntfile.js (hunk 1 above)
Comment (localjo) [reply Gruntfile.js]: I would also check at the top, right after this line if a local options directory exists;
`var hasOptionsDirectory = fs.existsSync('options')`

File: Gruntfile.js [hunk 3]
```diff
@@ -45,16 +45,28 @@ module.exports = function(grunt) {
...
     release: 0
   };
 
-  var pkg = grunt.file.readJSON("package.json");
-
   if (fs.existsSync("options/version.json")) {
     options = grunt.file.readJSON("options/version.json");
   }
+  else if (fs.existsSync("node_modules/worldview-options-eosdis/version.json")) {
+    options = grunt.file.readJSON("node_modules/worldview-options-eosdis/version.json");
```
Comment (localjo) [inline Gruntfile.js]: Then you can replace lines 59-69 with;
```
var options = hasOptionsDirectory
  ? grunt.file.readJSON('options/version.json')
  : grunt.file.readJSON('node_modules/worldview-options-eosdis/version.json');
```

File: README.md [hunk 4]
```diff
@@ -103,6 +94,21 @@ http://localhost:3000
 A node server will continue running until you end the session.
 You can end the session by pressing `control-C`
 
+
+### Additional Configuration Options
+
+The [Official EOSDIS configurations](https://github.com/nasa-gibs/worldview-options-eosdis) are installed by default.   
+To use custom options, clone the template repository into an options directory in the root using the following git command.   
+Optionally, you can clone the official options and make changes as needed.
```
Comment (localjo) [inline README.md]: I'd move this line in between the two `git clone` examples, since it refers to the latter.

File: Gruntfile.js (hunk 1 above)
Comment (ZachTRice) [reply Gruntfile.js]: Awesome. Didn't know you could use `require` in that situation.

File: README.md (hunk 4 above)
Comment (ZachTRice) [reply README.md]: Sure, I can do that

**Comment by ZachTRice** [comment]
The options-check branch can be merged into development now. If you want to test it out, pull down the branch, remove your options folder, run npm install, and then grunt. This will pull options (v2.1.2) from git. To override this options repo, use the old method of cloning the options into an options folder.

**Pull request merged**